src/CocktailBar.Domain/Common/Base/Classes/AggregateRoot.cs

```diff
@@ -22,4 +22,6 @@ public abstract class AggregateRoot<TId> : EntityWithMetadata<TId>
     protected AggregateRoot(TId id) : base(id)
     {
     }
+
+    protected AggregateRoot() { }
```

Comment:
Should probably make the Create static method here or inside the interface if possible.